_posts/2020-09-28-waku-v2-update.md

```diff
@@ -1,12 +1,12 @@
 ---
```

Comment:
I notice assets/img/coscup-waku/walletconnect.png and some other files are deleted - perhaps this is the latest content that hasn't been incorporated?